spec/index.bs

```diff
@@ -401,17 +401,17 @@ interface GPUColorWriteBits {
 };
 
 dictionary GPUBlendDescriptor {
-    GPUBlendFactor srcFactor;
-    GPUBlendFactor dstFactor;
-    GPUBlendOperation operation;
+    GPUBlendFactor srcFactor = "one";
+    GPUBlendFactor dstFactor = "zero";
+    GPUBlendOperation operation = "add";
 };
 
 dictionary GPUColorStateDescriptor {
-    GPUTextureFormat format;
+    required GPUTextureFormat format;
 
-    GPUBlendDescriptor alphaBlend;
-    GPUBlendDescriptor colorBlend;
-    GPUColorWriteFlags writeMask;
+    required GPUBlendDescriptor alphaBlend;
```

Comment:
What does it mean for a required field that all has default values? Can you omit it or are you required to do `colorBlend: {}, alphaBlend: {}` ?

Reply:
We have to write it this way for now because IDL doesn't support default values for members with dictionary-type. We can work around this in the spec by specifying what "undefined" means for these members.

Reply:
Right, we should probably make it optional with a default value of `null` and prose in the spec saying what `null` means.

Reply:
Right. (After this PR of course.)

Reply:
Ah, thanks for the quick answer. Yep, agreed.